subsys/bluetooth/controller/hci_driver.c

```diff
@@ -278,7 +279,13 @@ static bool fetch_and_process_hci_evt(uint8_t *p_hci_buffer)
 
 	errcode = MULTITHREADING_LOCK_ACQUIRE();
 	if (!errcode) {
-		errcode = sdc_hci_evt_get(p_hci_buffer);
+		/* First try fetching a command complete or a command status. */
```

Comment:
Would it be better adding a function to hci_cmd_put for this. Therefore, the expose API of hci_cmd_put.h becomes hci_cmd_put() and hci_evt_get()?

Reply:
I agree.

- [ ]  Move out hci_evt get and rename hci_cmd_put.c to hci.